control-plane/p4RuntimeSerializer.cpp

```diff
@@ -639,6 +640,78 @@ getMatchType(cstring matchTypeName) {
     }
 }
 
+/*
+ * This function supports a deeply nested mix of Type_Newtype and typedef.
+ *The function also supported Type_Bits.
+*/
+static uint32_t
+getTypeWidth(const IR::Type* type, TypeMap* typeMap, ReferenceMap* refMap) {
```

Comment:
I thought there was agreement to use Mihai's new typeMap method for this?

Reply:
Mihai's pass is delayed getting checked in.  I can add a TODO note above the function to say "integrate with sizeof PR when it gets checked in".   Right now, I am not positive if Mihai's PR will solve all cases of daisy-chains to `typedef` and `type` which is supported here.

Reply:
@antoninbas I patched in Mihai's PR code changes and the new typeMap method works for my PR.  If there aren't any issues, please approve this PR for p4runtime changes to p4c.